src/g3log.cpp

```diff
@@ -153,17 +153,16 @@ namespace g3 {
 
 
 
+      void saveMessage(const char *entry, const wchar_t *wentry, const char *file, int line, const char *function, const LEVELS &level,
+         const char *boolean_expression, int fatal_signal, const char *stack_trace) {
 
-      /** explicits copy of all input. This is makes it possibly to use g3log across dynamically loaded libraries
-      * i.e. (dlopen + dlsym)  */
-      void saveMessage(const char *entry, const char *file, int line, const char *function, const LEVELS &level,
-                       const char *boolean_expression, int fatal_signal, const char *stack_trace) {
          LEVELS msgLevel {level};
          LogMessagePtr message {std2::make_unique<LogMessage>(file, line, function, msgLevel)};
+
+         message.get()->wwrite().append(wentry);
```

Comment:
The reason why I'm mentioning this is that it might be a platform dependent issue to write ```nullptr```